vizro-core/src/vizro/_vizro.py

```diff
@@ -13,6 +13,18 @@
 logger = logging.getLogger(__name__)
 
 
+def _clear_state():
```

Comment:
Shall we actually add this as a private method into the Vizro class? Then people can just call `Vizro._clear_state()` and don't have to import the private method via the private file etc.?

```suggestion
def _clear_state():
```

Reply:
Hm I think that may ultimately be the better option, let me implement